clients/spark/core/src/main/scala/io/treeverse/clients/GCBackupAndRestore.scala

```diff
@@ -78,16 +82,52 @@ object GCBackupAndRestore {
     }
   }
 
+
+
+  // control copy speed
+  //  public static final String CONF_LABEL_DIRECT_WRITE = "distcp.direct.write";
+
   // Construct a DistCp command of the form:
   // `hadoop distcp -D<per storage credentials> -f absoluteAddressesTextFilePath dstNamespaceForHadoopFs`
   // example for command format https://docs.lakefs.io/integrations/distcp.html
+  // with distCp options from https://hadoop.apache.org/docs/r3.2.1/hadoop-distcp/DistCp.html#Command_Line_Options
   def constructDistCpCommand(
-      hadoopProps: Array[(String, String)],
-      absoluteAddressesTextFilePath: String,
-      dstNamespaceForHadoopFs: String
-  ): Array[String] = {
+                              hadoopProps: Array[(String, String)],
+                              absoluteAddressesTextFilePath: String,
+                              dstNamespaceForHadoopFs: String,
+                              hc: Configuration,
+                              numObjectsToCopy: Long
+                            ): Array[String] = {
+
+    val distCpLogsPath = hc.get(DistCpConstants.CONF_LABEL_LOG_PATH, dstNamespaceForHadoopFs + "_distCp/logs/")
+    print("distCpLogsPath: " + distCpLogsPath + "\n")
```

Comment:
```suggestion
    println("distCpLogsPath: " + distCpLogsPath)
```
or
```suggestion
    println(s"distCpLogsPath ${distCpLogsPath}")
```
(and similarly below).

Also consider first fetching all the numbers, then  outputting them in one go.